product-service/serverless.ts

```diff
@@ -37,6 +37,73 @@ const serverlessConfiguration: Serverless = {
       DB_NAME: process.env.DB_NAME,
       DB_USERNAME: process.env.DB_USERNAME,
       DB_PASSWORD: process.env.DB_PASSWORD,
+      IMPORT_PRODUCT_SNS_TOPIC_ARN: {
+        Ref: 'importProductSnsTopic'
+      }
+    },
+    iamRoleStatements: [
+      {
+        Effect: 'Allow',
+        Action: 'sqs:*',
+        Resource: {
+          'Fn::GetAtt': ['CatalogBatchQueue', 'Arn'],
+        },
+      },
+      {
+        Effect: 'Allow',
+        Action: 'sns:*',
+        Resource: {
+          Ref: 'importProductSnsTopic'
+        }
+      }
+    ],
+  },
+  resources: {
+    Resources: {
+      CatalogBatchQueue: {
+        Type: 'AWS::SQS::Queue',
+        Properties: {
+          QueueName: `${process.env.SQS_QUEUE_NAME}-${process.env.STAGE}`,
+          ReceiveMessageWaitTimeSeconds: 20,
+        }
+      },
+      importProductSnsTopic: {
+        Type: 'AWS::SNS::Topic',
+        Properties: {
+          TopicName: `${process.env.IMPORT_PRODUCT_SNS_TOPIC_NAME}-${process.env.STAGE}`,
+        }
+      },
+      importProductSnsSubscription: {
+        Type: 'AWS::SNS::Subscription',
+        Properties: {
+          Endpoint: 'vorobev.profi@gmail.com',
```

Comment:
This is sensitive information, therefore it's better to put it to env vars

Reply:
+1